Makefile

```diff
@@ -113,23 +113,33 @@ submodules:
 	@git submodule sync
 	@git submodule update --init --recursive
 
-.PHONY: manifests cobertura reviewable submodules fallthrough test-integration
+# This is for running out-of-cluster locally, and is for convenience. Running
+# this make target will print out the command which was used. For more control,
+# try running the binary directly with different arguments.
+run: go.build
+	@$(INFO) Running Crossplane locally out-of-cluster . . .
+	@# To see other arguments that can be provided, run the command with --help instead
+	$(GO_OUT_DIR)/$(PROJECT_NAME) --debug
+
+.PHONY: manifests cobertura reviewable submodules fallthrough test-integration run
 
 # ====================================================================================
 # Special Targets
 
-define CROSSPLANE_HELP
+define CROSSPLANE_MAKE_HELP
 Crossplane Targets:
```

Comment:
should this new `run` target be added to the help output here?

Reply:
Good point; that seems like a good idea